### ISSUE TITLE
Error 500 after upgrading to v3.0.0

### ISSUE THREAD

**Issue by nickurt** [myparcelnl/woocommerce#121]
Got a update in WordPress to upgrade (from 2.x) to 3.0.0, got error

```
7026#0: *382570 FastCGI sent in stderr: "PHP message: PHP Warning: require_once([..]/httpdocs/wp-content/plugins/woocommerce-myparcel/includes/class-wcmp-frontend-settings.php): failed to open stream: No such file or directory in [..]/httpdocs/wp-content/plugins/woocommerce-myparcel/includes/class-wcmp-frontend.php on line 66

PHP message: PHP Fatal error: require_once(): Failed opening required '[..]/httpdocs/wp-content/plugins/woocommerce-myparcel/includes/class-wcmp-frontend-settings.php' (include_path='.:/opt/plesk/php/7.2/share/pear') in [..]/httpdocs/wp-content/plugins/woocommerce-myparcel/includes/class-wcmp-frontend.php on line 66" while reading response header from upstream
```

Removed the plugin, tried installing it again, same error .. 

Any help?

**Comment by nickurt** [comment]
Downloaded the ZIP file from WordPress Plugins, it included every file, except the `class-wcmp-frontend-settings.php`

```
/includes/class-wcmp-admin.php
/includes/class-wcmp-api.php
/includes/class-wcmp-assets.php
/includes/class-wcmp-export.php
/includes/class-wcmp-frontend.php
/includes/class-wcmp-nlpostcode-fields.php
/includes/class-wcmp-rest.php
/includes/class-wcmp-settings-callbacks.php
/includes/class-wcmp-settings.php
```

Can you fix this @RichardPerdaan

**Comment by RichardPerdaan** [comment]
Hi @nickurt,

Thank you for your massage.
This problem is fixed in version 3.0.1 of the MyParcel plugin.
